### PR TITLE
Attendee care

### PR DESCRIPTION
**Pull request by Malwine**
@lislis and me met for yummi summer rolls and discussed the attendee care task. With this description anybody should be able to take it over without problems. Of course we are here to help.
:warning: There might be some typos. :smile: 

Please everybody feel welcome to comment/change etc.

@plexus You may want to have a look?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,9 +18,10 @@ https://clojurebridge-berlin.github.io/
 
 ### Past Events
 - [10. & 11. July 2015](http://www.clojurebridge.org/events/2015-07-10-berlin)
+- [22 & 23 January 2016](http://www.clojurebridge.org/events/2016-01-22-berlin)
 
 ### Upcoming Event
-- [22 & 23 January 2016](http://www.clojurebridge.org/events/2016-01-22-berlin)
+Coming soon. For updates check our [website](http://www.clojurebridge.org/).
```
Comment (vsmart) [inline README.md]: or http://clojurebridge-berlin.github.io

File: clojurebridge_attendee_care.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Attendee care recipe
+
+`This is our recepe to attendee care. The tasks can be split. Different people can take over parts.`
+
+**Selection:**
+
+1. In our GoogleDrive:
```
Comment (vsmart) [inline clojurebridge_attendee_care.md:9]: maybe 0: We have a Google account you can receive access to from previous organizers. There you will find attendee documents and email correspondence.

Comment (Malwine) [inline clojurebridge_attendee_care.md:9]: I thought about that! If you think it is reasonable I will add it.

File: clojurebridge_attendee_care.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+**At the event:**
+
+11. **Friday:** Do the check in, welcome everybody. See who didn't come. Make sure people feel welcome and have someone to talk too (especially in their language).
+
+12. **Saturday:** Welcome everybody. Check in again. Check who came. Rematch the groups if people don't show up.
+
+13. Wander around the groups. Make sure people feel good. If there are problems, technical or personal take action. Ask floating coaches to jump in or help out. Keep people updated and informed. Remind them to take breaks, keep hydrated and happy.
+
+14. Send a [feedback/wrap-up mail](https://github.com/clojurebridge-berlin/organization/blob/master/email_templates/all_mails_to_attendees.md#mail-4---feedback-and-wrapup) after the event.
```
Comment (vsmart) [inline clojurebridge_attendee_care.md]: I guess 14 is a new section "after the event"
- send out feedback email
- who accumulates/ compiles the feedback we get? is that also attendee care? 
  Also: "Coordinate with the rest of team to make sure attendees are informed of our project groups and Clj meet-ups.

**Comment by vsmart** [comment]
@lislis and @Malwine thank you for writing these up <3 I left some small comments

File: README.md (hunk 1 above)
Comment (Malwine) [inline README.md]: True - I just copied without looking :no_good:

File: clojurebridge_attendee_care.md (hunk 2 above)
Comment (vsmart) [inline clojurebridge_attendee_care.md:9]: Yup! Better to be too obvious than leave out info, in my mind :)

**Comment by Malwine** [comment]
Thanks @vsmart for reviewing. We will make changes as proposed.

File: clojurebridge_attendee_care.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+7. If somebody doesn't get back to any deadline you can send them [a mail](https://github.com/clojurebridge-berlin/organization/blob/master/email_templates/all_mails_to_attendees.md#to-applicants-who-didnt-get-back) that we gave their spot to the next person.
+
+**Shortly before the event:**
+
+8. When you gathered all the information who attends start to match the groups. It's important to match for language and experience. *We do have attendees who can't speak English. Please consider this carefully.* Also it makes sense to match people who don't want to be photographed.
+
+9. 5 Days before the event: Send another [mail with details about the event](https://github.com/clojurebridge-berlin/organization/blob/master/email_templates/all_mails_to_attendees.md#mail-3---information-before-the-event).
+
+10. Before the event: prepare name tags with name and photo preference badge.
```
Comment (vsmart) [inline clojurebridge_attendee_care.md:31]: Actually this was something I wanted to give feedback on and forgot. I think it's kind of nice for attendees to be able to customise their own name tag. I see the advantage of having someone prepared for everyone - especially with the photo preference sticker - but I think letting people pick what name they want their name tag to have is better. it's just a small thing but it's easy for us to do. In fact it is possibly less work for us this way! What do you think? 

EDIT: This PR is probably not the place to discuss this though! Sorry

Comment (Malwine) [inline clojurebridge_attendee_care.md:31]: Why don't you open an issue? Then we can keep track of this. I think that's a valid point to be discussed :)

Comment (Malwine) [inline clojurebridge_attendee_care.md:31]: I quoted you in the do different issue.

**Pull request merged**